src/components/Carousel.tsx

```diff
@@ -29,11 +43,20 @@ const Carousel: React.FC<Props> = ({ images }) => {
     }
   };
 
+  const containerWidth = visibleImages * inputValue;
+
   const translateValue = `translateX(-${currentIndex * imageSize}px)`;
 
+  const isPrevDisabled = currentIndex === 0;
+  const isNextDisabled = currentIndex + visibleImages === images.length;
```

Comment:
it might happen to be >, so 
```suggestion
const isNextDisabled = currentIndex + visibleImages >= images.length;
```